src/components/ebay-donut-chart/components/ebay-chart-legend/index.marko

```diff
@@ -0,0 +1,45 @@
+import type { DonutLegendItem } from "../../component";
+export interface Input {
+    focusIndex: number;
+    items: DonutLegendItem[];
+    "on-focus": (index: number) => void;
+    "on-blur": () => void;
+}
+class {
+    handleFocus(id: number) {
+        this.emit("focus", id);
+    }
+
+    handleBlur() {
+        this.emit("blur");
+    }
+}
+
+<div class="ebay-donut-chart__legend">
+    <for|item, index| of=input.items>
+        <div
+            class=[
+                "ebay-donut-chart__legend-item",
+                input.focusIndex !== -1 &&
+                    input.focusIndex !== index &&
+                    "ebay-donut-chart__legend-item--inactive",
+            ]
+            on-mouseover("handleFocus", index)
+            on-mouseout("handleBlur")
+            on-click("handleFocus", index)
+            on-focus("handleFocus", index)
+            on-blur("handleBlur")
+        >
+            <div class="ebay-donut-chart__legend-label">
+                <div class=[
+                    "ebay-donut-chart__legend-symbol",
+                    `ebay-donut-chart__legend-symbol--${item.symbolClass}`,
```

Comment:
Is this always defined?
If not, we might need to add a guard like
```item.symbolClass && `ebay-donut-chart__legend-symbol--${item.symbolClass}\`,```

Reply:
Good call